robotpy_ext/misc/periodic_filter.py

```diff
@@ -0,0 +1,13 @@
+class PeriodicFilter:
+    def __init__(self, parent):
+        '''
+        :type parent: Object
+        '''
+        
+        self.parent = parent
+        
+    def filter(self, record):
+        if self.parent.loggingLoop:
+            return True
+        else:
+            return record.levelno > 20
```

Comment:
This could be more simply expressed as
```python
def filter(self, record):
  return self.parent.loggingLoop or record.levelno > 20
```

What is this magic number `20` though?